### DISCUSSION TITLE
With QRZ lookup, no grid square.

### DISCUSSION THREAD

**Discussion by rc325** [wavelog/wavelog#690] [Q&A] [answered]
I'm using QRZ for lookup but get no gridsquares when logging a QSO--- I've tried a few different config variables in wavelog.php and config.php but they don't seem to influence the behavior.  I found one other  thread on this topic but the answers within didn't address a path forward server side.

Any suggestions on how to proceed? Appreciate any insight.

Best,
rc325

**Comment by violarulan** [comment, marked as answer]
You need to have XML Subscription.

If you do have one, provide the API Key in the Station Setup page.

**Reply by rc325** [reply]
Thank you. Have you considered adding an API like HamDB for lookup?

Appreciate all the work on this. 
Best,
RC325

**Reply by HB9HIL** [reply]
Hello @rc325 

We do support qrz.com and hamqth. As free solution you can use hamqth or qrz.com without a subscription. Even though you'll get limited data.

We can recommend the XML subscription by qrz.com.

Importantnto say that we, the Wavelog Team, have no power here and it isn't our decision which data comes back (and which not without subscription).

**Reply by rc325** [reply]
Totally understand, the information is controlled by the upstream provider. I guess reading between the lines, you don't have an intent to support other upstream data services?

Thanks for your time. 

Best,
RC325

**Reply by HB9HIL** [reply]
Currently we do not plan to support other callbooks. Sorry.
